package/upgrade/upgrade_manifests.sh

```diff
@@ -450,6 +450,17 @@ pause_all_charts()
   done
 }
 
+upgrade_addons()
+{
+ if test $UPGRADE_PREVIOUS_VERSION < 1.1.0; then
```

Comment:
Here we have a mistake, please update:

`if test "$UPGRADE_PREVIOUS_VERSION" \< "1.1.0"; then `
or
`if test "$UPGRADE_PREVIOUS_VERSION" \< 1.1.0; then `

bash needs `\<` to represent "<"

Reply:
And be aware this only works for formal releases. Comparison like `test "v1.1.0-rc1" \< "v1.1.0"` is false.
I guess we can keep this one.

Reply:
I have changed the logic slightly to make it easier to compare versions

Reply:
@bk201   the `v1.1.0` is  < `v1.1.0-rc1`

```
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
..
>>> "v1.1.0" < "v1.1.0-rc1"
True

```

Reply:
No. But we can have a better way to check symvers. Does `lower_version_check` work here?

Reply:
![image](https://user-images.githubusercontent.com/31133476/194016523-47067692-0798-42c2-824c-75005ca736f3.png)